compile.sh

```diff
@@ -13,7 +13,13 @@ else
     ANDROID_ROOT=$(realpath "$ANDROID_BUILD_TOP")
 fi
 
-_out=$ANDROID_ROOT/out/kernel-$_kernel_major$_kernel_minor/$_compiler/$_device
+if [ ! -z $_separate_kernel_dir ]; then
```

Comment:
Hmm, for `_fastboot_flash` we explicitly check for `true`:

https://github.com/MarijnS95/oot/blob/5bc615338d22272bc585e03b6cff719716bc5693/compile.sh#L65

While this works, it'll confuse those who'd write `=false` during some odd experiment and be surprised ;)